albumentations/augmentations/functional.py

```diff
@@ -1512,10 +1512,10 @@ def fancy_pca(img, alpha=0.1):
     return orig_img
 
 
-@clipped
 def glass_blur(img, sigma, max_delta, iterations, dxy, mode):
     coef = MAX_VALUES_BY_DTYPE[img.dtype]
-    x = np.uint8(cv2.GaussianBlur(np.array(img) / coef, sigmaX=sigma, ksize=(0, 0)) * coef)
+    x = cv2.GaussianBlur(np.array(img) / coef, sigmaX=sigma, ksize=(0, 0)) * coef
```

Comment:
Can you remove normalization by coef?
With this changes we also don't need normalize `x` in the and and clip it

Reply:
@Dipet  OK, I see your point.

Can I ask for some advice about the test case?
In the first PR version, I added a test case to check the transformed values of uint8, float32 images are approximately the same.
My intention against the test case is to confirm some dtype operations are done appropriately.
Thanks to your review, now the code has no explicit dtype operations, so I think the test case becomes less effective.
Is it better to remove my test case?
If there is any idea about this issue’s test case, let me know?

Reply:
Fixed.

Reply:
I think it is good that we have this test. 
In my opinion all transforms in the library must have similar tests to check that transforms work the same for uint8 and float datatypes.

Reply:
OK, I see. Thanks for the comment.